### PR TITLE
442 - Tweaks to contribution page banner

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number
Closing #442

## Purpose/Implementation Notes
Made minor tweaks to contribution page banner. 

@dvenprasad please review the latest UI [here](https://scpca-portal-7h7glvvo1-ccdl.vercel.app).

## Types of changes
- Bugfix (non-breaking change which fixes an issue)

## Checklist
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: client/src/components/WarningText.js [hunk 1]
```diff
@@ -21,7 +22,11 @@ export const WarningText = ({
       margin={{ top: 'small', bottom: 'medium' }}
     >
       <Box margin={iconMargin} pad={iconPad}>
-        <Icon color={iconColor} size={iconSize} name="Warning" />
+        <Icon
+          color={iconColor}
+          size={iconSize}
+          name={`Warning${iconNoFill ? 'NoFill' : ''}`}
```
Comment (davidsmejia) [inline client/src/components/WarningText.js:28]: Can we just have a prop called `iconName` and default it to `WarningNoFill`. Would provide more flexibility and I'm pretty sure we will always use the "no fill" option.

**Review by davidsmejia — CHANGES_REQUESTED** [review]
Small change requested looks good though

File: client/src/components/WarningText.js (hunk 1 above)
Comment (davidsmejia) [reply client/src/components/WarningText.js:28]: Ah ok, So in that case we should just switch to `iconName` and default it to `Warning` and then for this one use case we pass in `WarningNoFilll`. We won't have to update the other uses of this component and we can now expand it as we need for new iconography.

File: client/src/images/warning-red.svg
Comment (davidsmejia) [inline client/src/images/warning-red.svg:1]: Can we keep the icon file name and icon name matching? So, either rename this to `warning-no-fill` or change the name to be `WarningRed` in `components/icon` to match this file name.

Comment (nozomione) [reply client/src/images/warning-red.svg:1]: The icon's color is set dynamically using the `iconColor` prop, so I updated the name of the image file to be `warning-no-fill`. Please review,. thank you David.

**Review by davidsmejia — APPROVED** [review]
LGTM

**Pull request merged**